server/app.ts

```diff
@@ -0,0 +1,21 @@
+import express, { Request, Response } from 'express';
+import compression from 'compression';
+import morgan from 'morgan';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+
+export const app = express();
```

Comment:
we should specify the type of app maybe we use 
 const app: Application = express();
or 
const app: Express = express();
I don't know what is better